VARIABLES.md

```diff
@@ -16,6 +16,7 @@
 | apiary\_log\_prefix | Prefix for Apiary logs. | `string` | `""` | no |
 | apiary\_managed\_schemas | List of maps, each map contains schema name from which S3 bucket names will be derived, and various properties. The corresponding S3 bucket will be named as apiary\_instance-aws\_account-aws\_region-schema\_name. | `list(map(string))` | `[]` | no |
 | apiary\_producer\_iamroles | AWS IAM roles allowed write access to managed Apiary S3 buckets. | `map(any)` | `{}` | no |
+| apiary\_governance\_iamroles | AWS IAM governance roles allowed read and tagging access to managed Apiary S3 buckets. | `list(string)` | `[]` | no |
```

Comment:
I hate to be nitpicky, but we try to keep these in alphabetical order.  Can you move this line to just after line 14?

Reply:
Done